test/integration/bad/missing_dist_suffix/non_existing_distribution_suffix4.stan

```diff
@@ -0,0 +1,5 @@
+data { 
+}
+model {  
+    target += von_mises_notasuffix(1|0,1);
```

Comment:
```suggestion
    target += von_mises_notasuffix(1,0,1);
```

This is testing a different error message which involves the use of `|`, we should try to trigger the new behavior (currently this change leads to the test outputting the old "An undecleared identifier" message